build/dsc/templates/dsc_type.rb.erb

```diff
@@ -4,6 +4,7 @@ Puppet::Type.newtype(:dsc_<%= resource.friendlyname.downcase %>) do
   require Pathname.new(__FILE__).dirname + '../../' + 'puppet/type/base_dsc'
 
   provide :powershell, :parent => Puppet::Type.type(:base_dsc).provider(:powershell) do
+    confine :operatingsystem => :windows, :powershell_version => '5.0.10514.6'
```

Comment:
Yeah, Windows 10.. sigh.  Thanks for the reminder.  Did you happen to check LCM behavior with `Invoke-DscResource` on that version?

So we have `5.0.10514.6`, unless on Windows 10... in which case we have `5.0.10240.16384`.  And no Windows Updates have shown up yet for it either?

I wonder if our best bet is to:

* Confine to `5.0.10240.16384` or higher (for now)
* Emit a warning from Puppet about the LCM business if < `5.0.10514.6` (for now)
* Add a small note to README (for now)

Then hopefully we can change the confine to `5.0.10514.6` pretty soon, and update the README by the time the module hits 1.0 / supported.

Thoughts @jpogran / @ferventcoder ?